datadog_lambda/tracing.py

```diff
@@ -226,6 +257,9 @@ def extract_context_from_sqs_or_sns_event_or_context(event, lambda_context):
 
     try:
         first_record = event.get("Records")[0]
+        arn = first_record.get("eventSourceARN", "")
+        if arn:
+            is_sqs = True
 
         # logic to deal with SNS => SQS event
         if "body" in first_record:
```

Comment:
You do not currently have any testing for this block of code. Make sure to test the SNS => SQS case.